lib/updates.js

```diff
@@ -101,6 +101,8 @@ exports.apply = function(callback) {
 							if (update.__commit__ !== false) {
 								new Update({ key: file }).save();
 							}
+						} else {
+							done(err);
```

Comment:
This doesn't seem right either, since `done` is called on L108 anyhow, i.e. if an error occurs it will call `done` for a second time...